README.md

```diff
@@ -37,6 +37,34 @@ COPY your-application.jar app.jar
 You can then build the docker container using `docker build` or [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/).
 By default, the CMD is set to run the application JAR. You can change this by specifying your own `CMD` or `ENTRYPOINT`.
 
+### Container Memory Limits
+In order to have an optimal usage of the container memory you can indicate a memory limit through the [Downward API](https://kubernetes.io/docs/tasks/configure-pod-container/environment-variable-expose-pod-information).
```

Comment:
Can you change the first sentence to say: "To help the runtime compute accurate JVM memory defaults when running on Kubernetes, you can indicate memory limit...."?